### PR TITLE
Add styling

### PR DESCRIPTION
**Pull request by diarisdiakite**
In this PR, I:
✅ Added the missing JSX elements to my project (without adding extra functionality)
✅ Styled the project's final design to match Zeplin's design (this includes all elements + the same styling)
✅ Used the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/assets/css/books.module.scss [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+
+.blue {
+  color: $blue;
+  font-weight: 300;
+  font-size: small;
+  margin-bottom: 5%;
+}
+
+.img {
+  width: auto;
```
Comment (ademibowale) [inline src/assets/css/books.module.scss]: - [ ] Hey good job you did a great job on this milestone stone but there are some issues you still need to address 
- [ ] I would suggest you stick to the original given design, your Styling should be  EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/), kindly  style the nav bar to fit the given design 
- [ ] see video screensrec below 

https://github.com/diarisdiakite/bookstore/assets/92458236/3da54b4f-9a37-47d4-8d03-89cb216ede1d

- [ ] given design
bookstore

File: src/assets/css/books.module.scss [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+          height: 1.125rem;
+          margin: 0 2.438rem 0 0;
+          opacity: 0.5;
+          font-size: 0.875rem;
+          font-weight: bold;
+          font-stretch: normal;
+          font-style: normal;
+          line-height: normal;
+          letter-spacing: normal;
+          color: var(--black-two);
```
Comment (ademibowale) [inline src/assets/css/books.module.scss:76]: - [ ] I would kindly suggest you to make your website to be more responsive to fit all screen size please fix this 
- [ ] see screenrec attached below

https://github.com/diarisdiakite/bookstore/assets/92458236/2f355bba-3871-4e13-ad4e-49ee7baa21da

**Review by ademibowale — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @diarisdiakite    😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

just a little bit

## Highlights
  ✅ Nice user interface
  ✅Correct use of Github Flow
  ✅ No linter errors
✅ Descriptive Pull Request
  

## Required Changes ♻️




_Check the comments under the review._




## Optional suggestions 
   
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @diarisdiakite 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

image


Highlights
- [x] Good project organization ✔️
- [x] Nice use of React features ✔️ 
- [x] Kudus on meeting the requirements suggested  ✔️

### Confirmed view:
image

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**